src/quotes/backEndQuotes.js

```diff
@@ -0,0 +1,5 @@
+const QUOTES = {
```

Comment:
I think in this case lowercase _quotes_ is more appropriate. Overall all caps constants are in most cases associated with C styled 1 liners.